### PR TITLE
Adds projects

### PR DESCRIPTION
**Pull request by vhf**
@mathewmeconry Here's for https://docs.bexio.com/resources/pr_project/ , inspired by Contacts.

Great lib btw!

### CODE REVIEW

**Comment by vhf** [comment]
@mathewmeconry I added the tests, missed some on my first try. Unfortunately I couldn't get the tests to run locally.

After setting up the required env vars, I keep hitting this exception: https://github.com/mathewmeconry/bexio/blob/e2aba46550bd9ef3280aa12f0553abc7ac8d9803/src/index.ts#L261

Logging `res.body` right before [this line](https://github.com/mathewmeconry/bexio/blob/e2aba46550bd9ef3280aa12f0553abc7ac8d9803/src/index.ts#L258), I see that the response contains the form but no sign of `SAML` in the HTML.

**Comment by mathewmeconry** [comment]
@vhf thanks a lot for the pull and the compliments :)
I'm currently not near a computer so I'll try to find time this weekend to check the pull and the tests

**Comment by mathewmeconry** [comment]
@vhf I missed some settings in circleci for the build of forked pull requests. Could you please trigger a new build with an empty commit?

File: src/tests/ProjectStatuses.test.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  it("show project statuses", async () => {
+    const showedEntry = await moduleToTest.show({}, projectStatusEntry.id);
+    expect(showedEntry.name).to.be.eq(projectStatusEntry.name);
+  });
+
+  it.skip("overwrite a project status (not implemented by Bexio yet)", async () => {
+    const overwritten = await moduleToTest.overwrite(
+      projectStatusEntry.id,
+      {
+        name: "test"
```
Comment (mathewmeconry) [inline src/tests/ProjectStatuses.test.ts]: please use a name which is unique for each action and test
you can find an example here: https://github.com/mathewmeconry/bexio/blob/e2aba46550bd9ef3280aa12f0553abc7ac8d9803/src/tests/ContactTypes.test.ts#L85

File: src/tests/ProjectStatuses.test.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      });
+    } catch (err) {
+      expect(err.message).to.be.eq("not implemented by Bexio yet");
+    }
+  });
+
+  it.skip("edit project status (not implemented by Bexio yet)", async () => {
+    const edited = await moduleToTest.edit(projectStatusEntry.id, {
+      name: "test"
```
Comment (mathewmeconry) [inline src/tests/ProjectStatuses.test.ts]: please use a name which is unique for each action and test
you can find an example here: https://github.com/mathewmeconry/bexio/blob/e2aba46550bd9ef3280aa12f0553abc7ac8d9803/src/tests/ContactTypes.test.ts#L85

File: src/tests/ProjectStatuses.test.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    }
+  });
+
+  it("list project statuses", async () => {
+    const list = await moduleToTest.list({});
+    expect(list.map(el => el.id)).includes(projectStatusEntry.id);
+  });
+
+  // search is currently broken on bexio side
+  it.skip("search project statuses", async () => {
```
Comment (vhf) [inline src/tests/ProjectStatuses.test.ts]: This test just won't pass. 😞

File: src/tests/ProjectTypes.test.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    }
+  });
+
+  it("list project types", async () => {
+    const list = await moduleToTest.list({});
+    expect(list.map(el => el.id)).includes(projectType.id);
+  });
+
+  // search is currently broken on bexio side
+  it.skip("search project type", async () => {
```
Comment (vhf) [inline src/tests/ProjectTypes.test.ts]: Same here, apparently broken.

**Comment by vhf** [comment]
@mathewmeconry Hopefully everything is good now :)

**Comment by mathewmeconry** [comment]
your changes are now published with version 0.6.0

**Pull request merged**